docs/data-sources/repository/local/debian_repository.md

```diff
@@ -0,0 +1,31 @@
+---
+subcategory: "Local Repositories"
+---
+
+# Artifactory Local Debian Repository Data Source
+
+Retrieves a local Debian repository.
+
+## Example Usage
+
+```hcl
+data "artifactory_local_debian_repository" "my-debian-repo" {
```

Comment:
Same here with the `my-` prefix. Let's be consistent among our own documentation. I have no strong opinion on what prefix we want to use. `my-` is fine. I just want consistency.

Reply:
Sounds good, no my